samples/04_gis_analysts_data_scientists/export_video_using_arcgis_api_for_python.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>Add a narrative explaining what the video is showing and calling out the users attention to the shrinking lake. Also, can we increase the font size for the time?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Sure will do that

Reply:
<p>done</p>